src/System.IO.Compression/src/System/IO/Compression/DeflateStream.cs

```diff
@@ -209,6 +209,14 @@ public override void Flush()
             return;
         }
 
+        public override Task FlushAsync(CancellationToken cancellationToken)
+        {
+            EnsureNotDisposed();
```

Comment:
Here's what I wrote in the commit description:

> This commit just adds a FlushAsync override that returns an already completed task.  There's a slight potential behavioral change here, in that now if FlushAsync is called on a disposed object, the FlushAsync call will synchronously throw the ObjectDisposedException, whereas previously the Task returned from FlushAsync would be faulted with ObjectDisposedException.  I think this is the better behavior, as the ODE highlights a misuse of the object; if this is problematic, though, we can remove or ammend this commit.

In addition to that both ReadAsync and WriteAsync throw the ODE synchronously rather than asynchronously, so there's a consistency argument, too.